modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/types.ts

```diff
@@ -232,7 +232,7 @@ export interface IListSchema {
 
 export type ISchema = ITableSchema | ICardSchema | IListSchema;
 
-export type TViews = {
+export interface IView {
```

Comment:
Prefer interface over type.

Reply:
do we have a pattern for this? I've seen both indistinctly